tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -64,7 +67,10 @@ setup() {
 }
 
 @test "Cannot get CDH resource when deny-all policy is set" {
-	kbs_set_deny_all_resources
+	if ! is_confidential_hardware; then
+		kbs_set_deny_all_resources
+	fi
```

Comment:
I think `kbs_set_deny_all_resources` should still be called for all platforms. Otherwise I don't think this test will pass.

Reply:
Very good catch!